deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -2519,8 +2520,8 @@ NAN_METHOD(WebGLRenderingContext::GetProgramParameter) {
 
 
 NAN_METHOD(WebGLRenderingContext::GetUniformLocation) {
-  GLint programId = info[0]->ToObject()->Get(JS_STR("id"))->Int32Value();
-  v8::String::Utf8Value name(info[1]);
+  GLint programId = TO_INT32(JS_OBJ(info[0])->Get(JS_STR("id")));
+  Nan::Utf8String name(info[1]);
```

Comment:
inconsistent with the rest of the Nan::Utf8String lines (should `info[1]` be casted to a string here?)